src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -1431,40 +1676,101 @@
         }
       }
     },
-    "Comment": {
+    "CashAdvance": {
       "type": "object",
       "properties": {
-        "author": {
-          "description": "Author of the comment",
-          "$ref": "#/definitions/Employee"
+        "amountRequested": {
+          "description": "The amount of Cash Advance, expressed in the currency of this Cash Advance",
+          "$ref": "#/definitions/Amount"
         },
-        "creationDateTime": {
+        "approvalStatus": {
+          "description": "The approval status of the Cash Advance",
+          "$ref": "#/definitions/CashAdvanceApprovalStatus"
+        },
+        "cashAdvanceId": {
           "type": "string",
-          "description": "Creation date of the comment (in the format YYYY-MM-DD HH:mm:ss.SSS)"
+          "description": "The unique identifier of the Cash Advance"
         },
-        "isLatest": {
-          "type": "boolean",
-          "example": false,
-          "description": "Is set to true if the comment was edited after the last workflow transition"
+        "comment": {
+          "type": "string",
+          "description": "Last comment of this Cash Advance"
         },
-        "value": {
+        "exchangeRate": {
+          "description": "The exchange rate transferring by the reimbursement currency of the employee at the time he created the Request",
+          "$ref": "#/definitions/ExchangeRate"
+        },
+        "issueDate": {
           "type": "string",
-          "description": "Comment value"
+          "description": "The date when the Cash Advance was issued"
+        },
+        "requestDate": {
+          "type": "string",
+          "description": "The date when the Cash Advance was submitted (last submit date in case of recall)"
         }
       }
     },
-    "CustomField": {
+    "CashAdvanceApprovalStatus": {
       "type": "object",
       "properties": {
         "code": {
           "type": "string",
-          "description": "The short code of the list item. For non-list fields, this value will be blank"
-        },
-        "href": {
-          "type": "string",
-          "description": "The HATEOAS link to get this list item in case of a list. This value will be blank otherwise."
+          "description": "The code of the approval status of the Cash Advance",
+          "minLength": 1,
+          "maxLength": 2147483647,
+          "enum": [
+            "C_APPR",
+            "C_COMP",
+            "C_FILE",
+            "C_ISSU",
+            "C_NISU",
+            "C_NOTF",
+            "C_PECA",
+            "C_PEND",
+            "C_REJE"
+          ]
         },
-        "value": {
+        "name": {
+          "type": "string",
+          "description": "The approval status of the Cash Advance in English",
```

Comment:
The markdown states that this will be in the current user's language, not only in English. Which one is correct?